bindings/lua/Cargo.toml

```diff
@@ -29,9 +29,31 @@ rust-version.workspace = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+
+[features]
+default = ["mlua/lua52"]
+lua52 = ["mlua", "mlua/lua52"]
+
+# todo: fix mlua multi-version in `cargo build --all-features` error
+# https://github.com/apache/incubator-opendal/pull/2558#issuecomment-1613570958
+
+#lua54 = ["mlua", "mlua/lua54"]
+#lua53 = ["mlua", "mlua/lua53"]
+#lua52 = ["mlua", "mlua/lua52"]
+#lua51 = ["mlua", "mlua/lua51"]
+#luajit = ["mlua", "mlua/luajit"]
+#vendored = ["mlua", "mlua/vendored"]
+
+# for disable default features
+[dependencies.mlua]
```

Comment:
I prefer to write them in `dependencies` directly.

```
[dependencies]
mlua = { ... }
```

Reply:
it seems we can not write them in dependencies because `luarocks-build-rust-mlua` must use `lua52` features.

Reply:
The following is exactly the same.

```toml
[dependencies]
mula = { version = "0.8", features = ["module"], default-features = false, optional = true }
```